internal/cloudtools/installation.go

```diff
@@ -1,39 +1,34 @@
 package cloudtools
 
 import (
-	"fmt"
-
-	cloudModel "github.com/mattermost/mattermost-cloud/model"
+	cloud "github.com/mattermost/mattermost-cloud/model"
+	"github.com/pkg/errors"
```

Comment:
Archived for three years now. At some point we will need to start migrating codebases to `fmt.Errorf("%w", err)`